executor/builder.go

```diff
@@ -872,6 +872,7 @@ func (b *executorBuilder) buildShow(v *plannercore.PhysicalShow) exec.Executor {
 		Extended:              v.Extended,
 		Extractor:             v.Extractor,
 		ImportJobID:           v.ImportJobID,
+		FormatRaw:             v.FormatRaw,
```

Comment:
we can use `format = tidb_json` like analyze. I have changed the issue description.

Reply:
and the current name is so wide, better it as `PlacementFormat` or others.

Reply:
I am not sure, do you mean that?
- `raw` like now impl 
- `tidb_json` like a JSON format for origin rules